eng/packages/http-client-csharp-mgmt/emitter/test/resource-detection.test.ts

```diff
@@ -7,14 +7,12 @@ import {
 } from "./test-util.js";
 import { TestHost } from "@typespec/compiler/testing";
 import { createModel } from "@typespec/http-client-csharp";
-import { getAllClients, updateClients } from "../src/resource-detection.js";
+import { getAllClients, updateClients, buildArmProviderSchema } from "../src/resource-detection.js";
```

Comment:
I think in the new structure, we should not call `updateClients` here again to validate the applied decorators directly, why this import is still here?
Please clean it if it is not used, if it is still in use, please figure out if there is a way to replace such calls with `buildArmProviderSchema`.